### PR TITLE
DDF-1598: RrdJmxCollectorTest fails intermittently

### PR DESCRIPTION
**Pull request by tbatie**
- Added ignore flag to testRrdFileCreationForGaugeDataSource

@coyotesqrl @pklinef

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/293)

### CODE REVIEW

**Comment by coyotesqrl** [comment]
*LOOKS GOOD TO ME!* :100:

**Comment by coyotesqrl** [comment]
Pulled down branch and confirmed that this one test method is skipped and no others.

**Comment by roelens8** [comment]
:+1:

File: platform/metrics/platform-metrics-collector/src/test/java/ddf/metrics/collector/rrd4j/RrdJmxCollectorTest.java [hunk 1]
```diff
@@ -142,6 +143,7 @@ public void testRrdFileCreationForDeriveDataSource() throws Exception {
     }
 
     @Test
+    @Ignore
```
Comment (pklinef) [inline platform/metrics/platform-metrics-collector/src/test/java/ddf/metrics/collector/rrd4j/RrdJmxCollectorTest.java]: Please add a comment in the `Ignore` annotation about the problem this test is causing.

http://junit.sourceforge.net/javadoc/org/junit/Ignore.html

**Comment by tbatie** [comment]
Going to try to recreate the problem on bamboo with a verbose logger and extra test condition before finalizing the ignore annotation

**Pull request closed without merging**